File: src/main/webapp/resources/js/customer/platformAccount/plateformAccountService.js
//服务层
app.service('plateformAccountService',function($http){
	this.findById=function(id){
		return $http.get('../../../plateformAccount/findById?id='+id);
	}
	this.update=function(userEntity){
		return $http.post('../../plateformAccount/update',userEntity);
	}
	this.updateInfo=function(userEntity){
		return $http.post('../../plateformAccount/updateInfo',userEntity);
	}
	this.add=function(userEntity){
		return $http.post('../../plateformAccount/add',userEntity);
	}
	this.del=function(ids){
		return $http.get('../../plateformAccount/del?ids='+ids);
	}
	this.lock=function(id){
		return $http.get('../../plateformAccount/lock?id='+id);
	}
	this.unlock=function(id){
		return $http.get('../../plateformAccount/unlock?id='+id);
	}
	this.resetPassword=function(id){
		return $http.get('../../plateformAccount/resetPassword?id='+id);
	}
	this.findInfo=function(){
		return $http.get('../../plateformAccount/findInfo');
	}
	this.updatePass=function(userEntity){
		return $http.get('../../plateformAccount/updatePass?oldPassword='+userEntity.oldPassword+'&password='+userEntity.password);
	}
});

File: src/main/java/com/webside/plateformaccount/service/PlateformAccountService.java
package com.webside.plateformaccount.service;

import java.util.List;

import com.webside.plateformaccount.model.UserEntity;
import com.webside.util.JsonResult;
import com.webside.util.PageData;
import com.webside.util.PageResult;

public interface PlateformAccountService {
	public PageResult<List<UserEntity>> findPage(PageData pd);
	public void add(UserEntity userEntity);
	public UserEntity findById(Integer id);
	public UserEntity findByName(String name);
	public void update(UserEntity userEntity);
	public void delete(int[] ids);
	public void updateLock(Integer id);
	public void updateUnlock(Integer id);
	public JsonResult updateResetPassword(Integer id,String password);
	public JsonResult findInfo(PageData pd);
	public void updateInfo(UserEntity userEntity);
	public JsonResult updatePass(PageData pd);
}
